### DISCUSSION TITLE
Aiuto scelta sensori qualità aria per monitoraggio e sostituzione termostati stanze

### DISCUSSION THREAD

**Discussion by Master-of-Mouse** [indomus/forum#3311] [Hardware & Firmware]
Buongiorno, ho un sistema home assistant stabile che sta correttamente monitorando i consumi energetici con i sensori Shelly.
Ho le stanze ognuna con un vecchio termostato analogico (manopolina per la selezione della temperatura, 3 fili, due dell'alimentazione 220 ed uno che chiude il contatto di consenso per la caldaia) che cominciano a dare problemi dopo oltre 20 anni di servizio.

Anche per questioni di salute vorrei implementare dei sensori di qualità aria.
Dato che ne vorrei prendere 4 vorrei prenderne di buoni ma economici... diciamo con un ottimo rapporto qualità prezzo.
Poi andrei ad affiancarli da altrettanti sensori shelly per sostituire gli attuali termostati.

I sensori vorrei che rilevassero i seguenti parametri:
1. Temperatura
2. Umidità
3. Co2
4. VOC - Formaldeide, varia roba cancerogena ed irritante
5. PM?
6. Concentrazione ossigeno?

Temperatura, umidità e Co2 credo sia il minimo sindacabile.
I VOC sono tanti, non ho capito se i sensori li misurano tutti indistitamente o alcuni li rilevano ed altri no. Come minimo vorrei rilevare quelli più pericolosi.
PM, servono? Ovvero ha senso misurarli o, tenendo sotto controllo i VOC, il valore di PM lascia un po' il tempo che trova? O magari aiutano il sensore a dare un "voto" alla qualità aria della zona controllata?
Concentrazione ossigeno... credo che sia un parametro importante ma non lo vedo monitorato da nessuno. Non so se perchè non ha senso dato che monitorando gli altri parametri sarebbe un po' ridondante o boh.

Se poi esistono dei sensori di qualità aria con relè tanto meglio così li adatto nelle scatolette 503 degli attuali termostati e mi risparmio altri componenti.

Prima di tutto devono costare il giusto (meglio poco) ed integrarsi bene con home assistant. Non mi interessa il cloud, meno hanno a che fare con amazon, facebook e "spioni" vari meglio è. Un plus se hanno il relè per attivare direttamente la caldaia, altrimenti pazienza, ci penso in home assistant ad abbinare gli switch shelly. Un altro plus se hanno un display per selezionare la temperatura in modo da essere indipendenti e bypassare eventuali indisponibilità di home assistant.

Conoscete dei prodotti che si allineino con le mie richieste?

**Comment by NuccioT** [comment]
Io uso questo e da circa due anni nessun problema. L'unica cosa è che manda dati in continuazione.
https://www.zigbee2mqtt.io/devices/TS0601_air_quality_sensor.html#tuya-ts0601_air_quality_sensor
Acquistato su Aliexpress o Banggood

**Comment by KC-inDomus** [comment]
> 1. Temperatura
> 2. Umidità
> 3. Co2
> 4. VOC - Formaldeide, varia roba cancerogena ed irritante
> 5. PM?
> 6. Concentrazione ossigeno?

su questo ti consiglio l'[Amazon Sensor](https://indomus.it/componenti/amazon-smart-air-quality-monitor-multi-sensore-di-qualita-dellaria/)
